app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageViewModel.kt

```diff
@@ -74,16 +75,19 @@ class NewMessageViewModel : ViewModel() {
     }
 
     private fun configureDraft(draftUuid: String? = null) = viewModelScope.launch(Dispatchers.IO) {
-        val uuid = RealmDatabase.mailboxContent().writeBlocking {
-            return@writeBlocking if (draftUuid == null) {
+        RealmDatabase.mailboxContent().writeBlocking {
+            val uuid = if (draftUuid == null) {
                 createDraft()
             } else {
                 updateLiveData(draftUuid)
```

Comment:
We should rename this method because it doesn't speak for itself, especially since we don't see any `Livedata` here

Reply:
Will be fixed in #314.